ui/app/styles/helper-classes/layout.scss

```diff
@@ -77,7 +77,7 @@
 }
 
 .is-medium-width {
-  width: calc($desktop / 3);
+  width: calc($desktop / 2.5);
```

Comment:
longer formats went to a new line, so made the card a little wider (before `KEY ----` was a new line)
<img width="459" alt="Screenshot 2024-02-14 at 9 57 07 AM" src="https://github.com/hashicorp/vault/assets/68122737/feb27123-885d-4bfe-b3d7-670fab4f4d9a">